.github/workflows/auto-update-collector-version.yml

```diff
@@ -1,27 +0,0 @@
-name: Check collector version
-
-on:
-  workflow_dispatch:
-  schedule:
-    - cron: '0 1 * * *'
-
-jobs:
-  build:
-    # Make sure that the workflow does not run on forks (add yourself as exception if you work on the workflow)
-    if: github.repository_owner == 'open-telemetry' || github.repository_owner == 'svrnm'
-    name: Check collector version
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Auto-update
-        run: ./.github/workflows/scripts/check-collector-version.sh
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
If we're no longer using `GITHUB_TOKEN` (vs. `OPENTELEMETRYBOT_GITHUB_TOKEN ` in the new workflow), then `GITHUB_TOKEN` should be deleted from the settings, right?

Reply:
`GITHUB_TOKEN` is a built-in env var (not one that you add in your settings)